src/common/CryptoUtils.ts

```diff
@@ -132,6 +132,72 @@ export class CryptoUtils {
         };
     }
 
+    /**
+     * Converts a certificate from PEM to DER format.
+     *
+     * @param pemCertificate A string representing a certificate in PEM format.
+     * @returns A buffer containing the data of the certificate in DER format.
+     */
+    public static PEMtoDER(pemCertificate: string): Buffer {
```

Comment:
The method name is stretching the boundaries of proper lower camel case naming, which is the convention for JS/TS. I'd recommend `pemToDer()` or `pemToDER()`, but not the naming you have here.